src/types/local-settings.ts

```diff
@@ -1,12 +1,15 @@
 import { TableAgeType } from './table-age-type'
+import { SearchScope } from './searchScope'
 
 /**
  * This is where we store local state
  */
 export interface LocalSettings {
   ageHeaderType: TableAgeType
+  preferredScope: SearchScope | undefined
```

Comment:
Does pontus-x team really want this to be based on network selector clicks?

I would rename it to lastViewedScope and store it based on viewing history instead :shrug:

Reply:
Yes, according to their feedback, thie current behavior is exactly what they want.